CollapseLauncher/Classes/InstallManagement/StarRail/StarRailInstall.cs

```diff
@@ -21,6 +23,8 @@ internal class StarRailInstall : InstallManagerBase<GameTypeStarRailVersion>, IG
         #endregion
 
         #region Properties
+        // TODO: Need to look how the audio lang list works on HSR 1.5
```

Comment:
I believe the implementation is the same as GI currently, minus less voicepacks available iirc. Maybe a status update?

Reply:
It's "kinda" similar on how GI does it, minus they don't have the pkg_version for the voices and still serve the files via dispatch data too.
Since they are still migrating to this new method, it's kinda hard to say the status of it.

Reply:
Is it something that we want to implement for this version or it's still early in development?

Reply:
The current implementation should work, though we need to keep an eye on this since it might change on next update